js/compiler.ts

```diff
@@ -495,11 +509,71 @@ export class Compiler
       };
     });
   }
+}
 
-  private static _instance: Compiler | undefined;
+const compiler = new Compiler();
 
-  /** Returns the instance of `Compiler` or creates a new instance. */
-  static instance(): Compiler {
-    return Compiler._instance || (Compiler._instance = new Compiler());
+let startResMsg: msg.StartRes;
+
+/** Send to the privilaged side that we have setup and are ready. */
+function sendStart(): void {
+  const builder = flatbuffers.createBuilder();
+  msg.Start.startStart(builder);
+  const startOffset = msg.Start.endStart(builder);
+  const baseRes = sendSync(builder, msg.Any.Start, startOffset);
+  assert(baseRes != null);
+  assert(msg.Any.StartRes === baseRes!.innerType());
+  startResMsg = new msg.StartRes();
+  assert(baseRes!.inner(startResMsg) != null);
+}
```

Comment:
This seems to be copied from `js/main.ts` ... can you move this to `os.ts` so they can share the code?

Reply:
👍